docs/modules/release-notes/pages/all-releases/4_13_3S.adoc

```diff
@@ -0,0 +1,112 @@
+
+* The team assignment of test cases, test suites, and test runs is inherited from the baseline session from which they are created.
+* If the baseline session is created at the organization level, then test cases, test suites, and test runs created from it can only be accessed by users with the ADMIN predefined role who have selected the organization level from the team menu.
+* If the baseline session is created at the team level, then test cases, test suites, and test runs created from it can only be accessed by users assigned to that team who have selected that team from the team menu.
+
+[NOTE]
+====
+* Users with the ADMIN predefined role have access to all teams plus the organization level. To view test cases, test suites, and test runs of a specific team, select that team in the team menu.
+* Team assignment of test cases, test suites, and test cases cannot be changed.
+* Device bundles do not currently follow team assignments and can be accessed by all users.
+====
+
+=== Handling unsupported actions
+When there are actions that are unsupported by Scriptless in a test case:
+
+* Clear indications of the unsupported actions are displayed in test steps.
+* All steps containing unsupported actions must be removed from a test case before test runs can be triggered.
+
+=== Things to note:
+As this feature is in Beta, we welcome your feedback. Stay tuned for additional UI improvements and extended functionality in the coming months. We will publish a comprehensive User Guide on our documentation site soon. In the meantime, please note the following:
+
+* Launching Test Runs: test runs can be launched via:
+** The Rerun button on the test run results page.
+** The Run button in test case and test suite pages to make it easier to initiate a test run.
+** The Rerun button to the test run page. When selected, the configuration of the new test run will be pre-filled, including the specific device/device bundle previously used.
+
+* Testing Capabilities: With the Beta release, only crash and accessibility validations are available. We are actively developing performance, text/color, and visual validations. Data-driven testing is not supported in the Beta, but will be added in a future release.
+
+* Test Run Statuses:
+** Running: The test run session is either pending execution or in progress.
+** Needs Action: A blocker has been encountered and remediation is required.
+** Error: A system error has been encountered.
+** Success: The test run completed.
+** Failed: The application crashed during the test run.
+
+* Search and filters:
+** Filter test runs, test suites, and test cases by name or app bundle name. Note: Hit Enter after inputting the search query to start filtering.
+** Filter test suites and test runs by a single test case by selecting the number of test runs/test suites in the test case list

```

Comment:
Need a period at end to be consistent with first bullet. Or need to remove period from first bullet